purchase_work_acceptance/models/stock_picking.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class Picking(models.Model):
+    _inherit = "stock.picking"
+
+    wa_id = fields.Many2one(
+        comodel_name='work.acceptance',
+        string='WA Reference',
+        domain=[('state', '=', 'accept')],
+    )
+
+    @api.multi
+    def button_validate(self):
+        self.ensure_one()
+        if self.wa_id:
+            wa_lines = self.env['work.acceptance.line']
```

Comment:
Coding standard, always use camel case (capital letter), when declare Class.
```suggestion
            wa_lines = self.env['work.acceptance.line']
```